tiledb/sm/rest/rest_client_remote.cc

```diff
@@ -331,6 +327,47 @@ Status RestClientRemote::post_array_schema_to_rest(
   return sc;
 }
 
+void RestClientRemote::post_array_create_to_rest(
+    const URI& uri, const ArraySchema& array_schema) {
+  BufferList serialized{memory_tracker_};
+  auto& buff = serialized.emplace_buffer();
+
+  URI::RESTURIComponents rest_uri;
+  throw_if_not_ok(uri.get_rest_components(rest_legacy(), &rest_uri));
+  serialization::array_create_serialize(
+      array_schema, serialization_type_, buff, {rest_uri.asset_storage});
+
+  const auto creation_access_credentials_name{
+      config_->get<std::string>("rest.creation_access_credentials_name")};
+  if (creation_access_credentials_name.has_value()) {
+    add_header(
+        "X-TILEDB-CLOUD-ACCESS-CREDENTIALS-NAME",
+        creation_access_credentials_name.value());
+  }
```

Comment:
I don't think we need that in the new Server anymore

Reply:
Thanks, I didn't know that. How would the user override the default storage location?